wagtail/core/models.py

```diff
@@ -451,7 +451,8 @@ def clean(self):
     @transaction.atomic
     # ensure that changes are only committed when we have updated all descendant URL paths, to preserve consistency
     def save(self, *args, **kwargs):
```

Comment:
Could I suggest adding the argument to the method signature instead? I feel it would help visibility of this option.